extelasticache/common_elasticache.go

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2024 Steadybit GmbH
+
+package extelasticache
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/service/elasticache"
+	"github.com/steadybit/action-kit/go/action_kit_api/v2"
+	"github.com/steadybit/extension-aws/utils"
+	"github.com/steadybit/extension-kit/extutil"
+)
+
+const (
+	elasticacheNodeGroupTargetId = "com.steadybit.extension_aws.elasticache.node-group"
+)
+
+type ElasticacheClusterAttackState struct {
```

Comment:
Is this really a  "common" struct, as it's only used in the failover attack? Other functions in this file as well.

Reply:
both :D Nils ask me to create only this attack.. so we have only one action at the moment for elasticache..